### ISSUE TITLE
Is there any way to avoid overfitting while training over 200epoch?

### ISSUE THREAD

**Issue by Chanuku** [phillipi/pix2pix#202]
![overfitting](https://user-images.githubusercontent.com/15232937/93198523-1a796e00-f788-11ea-8b67-5f0a7eb51962.png)

as written on title, I'm finding for the hints to avoid overfitting while training over 200epoch

I tried to change learning rate, but it didn't work

**Comment by junyanz** [comment]
It seems that your learning rate has turned negative. This might cause the training failure.

**Comment by Chanuku** [comment]
Thanks for the reply,

If you know, can you give me any clue about which parameter should I fix to avoid this?

**Comment by junyanz** [comment]
You need to update your learning rate decay policy.
